docs/index.rst

```diff
@@ -69,6 +69,7 @@ If you use SMARTS in your research, please cite the `paper <https://arxiv.org/ab
    :caption: API
 
    api/modules.rst
+   sim/configuration.rst
```

Comment:
Maybe keep files under a given toctree in the same folder?

Reply:
It is hard to find where to put this since `api/*` is generated dynamically but environment variables and configuration files are related to `sim` but are relevant to API. Maybe not relevant enough. I am going to move it to resources.

Reply:
I am thinking `resources` is not quite the right spot for it either so I am moving it to next steps.